packages/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -312,6 +349,118 @@ const Calendar = WebexPlugin.extend({
         return Promise.all(promises).then(() => meetingObjects);
       });
   },
+
+  /**
+   * Create calendar event
+   * @param {object} [data] meeting payload data
+   * @returns {Promise} Resolves with creating calendar event response
+   * */
+  createCalendarEvent(data) {
+    return EncryptHelper.encryptCalendarEventRequest(this, data).then(() => {
+      return this.request({
+        method: 'POST',
+        service: 'calendar',
+        body: data,
+        resource: 'calendarEvents/sync',
+      });
+    });
+  },
+
+  /**
+   * Update calendar event
+   * @param {string} [id] calendar event id
+   * @param {object} [data] meeting payload data
+   * @param {object} [query] the query parameters for specific usage
+   * @returns {Promise} Resolves with updating calendar event response
+   * */
+  updateCalendarEvent(id, data, query) {
+    return EncryptHelper.encryptCalendarEventRequest(this, data).then(() => {
```

Comment:
unnecessary `return` statement, should be able to `.then(() => this.request({ /* ... */ });`

Reply:
@InteractiveTimmy it is done to simplify it